src/config.rs

```diff
@@ -252,6 +252,27 @@ fn get_total_memory_kb(root: &Path) -> Result<u64> {
     _get_total_memory_kb(&path)
 }
 
+fn _kernel_has_option(path: &Path, word: &str) -> Result<bool> {
+    let text = fs::read_to_string(path)?;
+
+    // The last argument wins, so check all words in turn.
+    let mut val = false;
```

Comment:
This is more obvious and doesn't allocate four times per each item
```rust
    // The last argument wins, so check all words in turn.
    Ok(text.split_whitespace().fold(false, |acc, w| {
        if !w.starts_with(word) {
            acc
        } else {
            match &w[word.len()..] {
                "" | "=1" | "=on" => true,
                "=0" | "=off" => false,
                _ => acc,
            }
        }
    }))
```

Reply:
I amended the patch with that.

Anyway, we need to wait for a reply from the anaconda folks whether they want this at all.